zammad/Chart.yaml

```diff
@@ -16,17 +16,21 @@ maintainers:
 dependencies:
   - name: elasticsearch
     repository: https://charts.bitnami.com/bitnami
-    version: 19.10.3
+    version: 19.17.0
     condition: zammadConfig.elasticsearch.enabled
   - name: memcached
-    version: 6.5.4
+    version: 6.9.0
     repository: https://charts.bitnami.com/bitnami
     condition: zammadConfig.memcached.enabled
   - name: postgresql
-    version: 12.6.4
+    version: 12.12.10
     repository: https://charts.bitnami.com/bitnami
     condition: zammadConfig.postgresql.enabled
   - name: redis
-    version: 17.11.7
+    version: 17.17.1
     repository: https://charts.bitnami.com/bitnami
     condition: zammadConfig.redis.enabled
+  - name: minio
```

Comment:
can we keep alphabetical sorting of dependencies please?

Reply:
Of course, I kept it in other places, but not here. Changed.